params/params.go

```diff
@@ -196,6 +196,7 @@ type Params struct {
 	// Special case: disable taxes with a value of 0
 	BlockTaxProportion uint16
 
+	// It must be hourglass block.
```

Comment:
Prerequisite of checkpoint not found from the PR. how to determine the checkpoint block type.